fuzzers/005-tilegrid/generate.py

```diff
@@ -110,25 +116,28 @@ def make_tile_baseaddrs(tiles, site_baseaddr, verbose=False):
     # Look up a base address by tile name
     tile_baseaddrs = dict()
 
-    verbose and print('')
-    verbose and print('%u tiles' % len(tiles))
+    verbose and print('', file=sys.stderr)
```

Comment:
I've never been a big fan of stderr personally.  FWIW there is a output file arg if you don't want the JSON to flood the screen